specification/recoveryservicesbackup/resource-manager/readme.md

```diff
@@ -45,6 +53,24 @@ model-validator: true
 semantic-validator: true
 message-format: json
 ```
+
+### Tag: package-passivestamp-2018-12-20
+These settings apply only when `--tag=package-passivestamp-2018-12-20` is specified on the command line.
+
+``` yaml $(tag) == 'package-passivestamp-2018-12-20'
+input-file:
+-  Microsoft.RecoveryServices/stable/2018-12-20/bms.json
+```
+
+### Tag: package-2021-08
+
+These settings apply only when `--tag=package-2021-08` is specified on the command line.
+
+```yaml $(tag) == 'package-2021-08'
+input-file:
+  - Microsoft.RecoveryServices/stable/2021-07-01/bms.json
```

Comment:
Should it be `2021-08-01`?

Reply:
Hi @qwordy. Since we have not made any changes to any APIs in this PR, we are going forward with the existing 2021-07-01 version for creating multiple packages.

Reply:
I will be pushing change to include the some s360 correctness related fixes in the new version 2021-08-01 and will update it by end of next week. Till then can we please hold the merging of this PR?

Reply:
@pratimaupadhyay02 OK